app/app.js

```diff
@@ -18,6 +20,19 @@ const getContainerId = () => {
     return containerId;
 };
 
+async function verifyPredatorVersion() {
+    if (semver.major(PREDATOR_RUNNER_VERSION) === semver.major(jobConfig.predatorVersion) &&
+        semver.minor(PREDATOR_RUNNER_VERSION) === semver.minor(jobConfig.predatorVersion)) {
+        return;
+    }
+    logger.error('Predator Runner and Predator must match in major and minor version, please change runner / predator version');
+    await reporterConnector.postStats(jobConfig, {
```

Comment:
no need for this, an error should be thrown and the errorHandler catches and will postStats

Reply:
I got no problem removing this, but people should upgrade their runner and predator at the same time IMO

Reply:
i agree regarding version upgrades
either way let's remove the 
```
    await reporterConnector.postStats(jobConfig, {
...
```
and throw an error instead so that this part will be handled with the errorHandler